openedx/core/djangoapps/schedules/templates/schedules/edx_ace/upgradereminder/email/body.html

```diff
@@ -0,0 +1,74 @@
+{% extends 'schedules/edx_ace/common/base_body.html' %}
+{% load i18n %}
+{% load static %}
+
+{% block preview_text %}
+    {% blocktrans trimmed %}
+        We hope you are enjoying learning with us so far in {{ course_name }}! A verified certificate
+        will allow you to highlight your new knowledge and skills. It's official, and easily shareable.
+
+        Upgrade by {{ user_schedule_upgrade_deadline_time }}.
+    {% endblocktrans %}
+{% endblock %}
+
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>{% trans "Upgrade now" %}</h1>
+
+            <p>
+                {% blocktrans trimmed %}
+                    We hope you are enjoying learning with us so far in <strong>{{ course_name }}</strong>! A
+                    verified certificate will allow you to highlight your new knowledge and skills. It's official,
+                    and easily shareable.
+                {% endblocktrans %}
+            </p>
+            <p>
+                {% blocktrans trimmed %}
+                    Upgrade by <strong>{{ user_schedule_upgrade_deadline_time }}</strong>.
+                {% endblocktrans %}
+            </p>
+
+            <a href="{{ course_url }}">
+                <img
+                    src="{% static "course_experience/images/verified-cert.png" %}"
```

Comment:
This resolves to `static/course_experience/images/verified-cert.png` on my devstack. I'd like to be sure that this will resolve to either https://courses.edx.org/static/course_experience/images/verified-cert.png or https://d37djvu3ytnwxt.cloudfront.net/static/course_experience/images/verified-cert.png in prod, but I don't know how to set up my settings locally to test that.

Reply:
Oh wait, that will never return an absolute url which is what we need here. I'm going to have to construct the url using `absolute_url` in the task and send it in the template context.